scripts/hudson/quickstart.sh

```diff
@@ -143,30 +143,6 @@ function build_narayana {
   rm -rf narayana
 }
 
-function configure_wildfly {
-  cd $WORKSPACE
-  wget -nv https://ci.wildfly.org/guestAuth/repository/downloadAll/WF_Nightly/.lastSuccessful/artifacts.zip
-  unzip -q artifacts.zip
-  # the artifacts.zip may be wrapping several zip files: artifacts.zip -> wildfly-latest-SNAPSHOT.zip -> wildfly-###-SNAPSHOT.zip
-  local wildflyLatestZipWrapper=$(ls wildfly-latest-*.zip | head -n 1)
-  if [ -f "${wildflyLatestZipWrapper}" ]; then # wrapper zip exists, let's unzip it to proceed further to distro zip
-    unzip -q "${wildflyLatestZipWrapper}"
-    [ $? -ne 0 ] && echo "Cannot unzip WildFly nightly build wrapper zip file '${wildflyLatestZipWrapper}'" && return 2
-    rm -f $wildflyLatestZipWrapper
-  fi
-  local wildflyDistZip=$(ls wildfly-*-SNAPSHOT.zip | head -n 1)
-  [ "x$wildflyDistZip" = "x" ] && echo "Cannot find any zip file of SNAPSHOT WildFly distribution in the nightly build artifacts" && return 1
-  unzip -q "${wildflyDistZip}"
-  [ $? -ne 0 ] && echo "Cannot unzip WildFly nightly build distribution zip file '${wildflyDistZip}'" && return 3
-  export JBOSS_HOME="${PWD}/${wildflyDistZip%.zip}"
-  [ ! -d "${JBOSS_HOME}" ] && echo "After unzipping the file '${wildflyDistZip}' the JBOSS_HOME directory at '${JBOSS_HOME}' does not exist" && return 4
-  cp $JBOSS_HOME/docs/examples/configs/standalone-xts.xml $JBOSS_HOME/standalone/configuration/
-  cp $JBOSS_HOME/docs/examples/configs/standalone-rts.xml $JBOSS_HOME/standalone/configuration/
```

Comment:
Do any of the quickstarts rely on standalone-rts.xml being copied over during this step?

Reply:
`standalone-rts.xml` is copied during the initialisation of JBOSS_HOME

Reply:
Thank you